File: king-admin-java/src/main/java/com/oukingtim/service/impl/TbTodoServiceImpl.java
package com.oukingtim.service.impl;

import com.baomidou.mybatisplus.service.impl.ServiceImpl;
import com.oukingtim.domain.TbTodo;
import com.oukingtim.mapper.TbTodoMapper;
import com.oukingtim.service.TbTodoService;
import org.springframework.stereotype.Service;

/**
 * Created by oukingtim
 */
@Service
public class TbTodoServiceImpl extends ServiceImpl<TbTodoMapper,TbTodo> implements TbTodoService {
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/service/impl/TbTodoServiceImpl.kt
package com.oukingtim.service.impl

import com.baomidou.mybatisplus.service.impl.ServiceImpl
import com.oukingtim.domain.TbTodo
import com.oukingtim.mapper.TbTodoMapper
import com.oukingtim.service.TbTodoService
import org.springframework.stereotype.Service

/**
 * Created by oukingtim
 */
@Service
class TbTodoServiceImpl : ServiceImpl<TbTodoMapper, TbTodo>(),TbTodoService